openreview/journal/journal.py

```diff
@@ -562,3 +565,13 @@ def notify_readers(self, edit, content_fields=[]):
 {content}
             '''
             self.client.post_message(recipients=[self.get_action_editors_id(number=forum.number)], subject=subject, message=message, ignoreRecipients=nonreaders, replyTo=self.contact_info)
+
+
+        if self.get_editors_in_chief_id() in readers and len(readers) == 2 and 'comment' in lower_formatted_invitation:
+            message = f'''Hi {{{{fullname}}}},
+
+{before_invitation} {lower_formatted_invitation} has been {action} on a submission that is only visible to you.
```

Comment:
I find the wording a bit confusing here, the comment is visible only to PCs correct, not the submission?

Reply:
Yes, I like: `An official comment has been posted on a submission for which you are serving as Editor-In-Chief.`